server/src/api/modules/name-changes/services/ApproveNameChangeService.ts

```diff
@@ -67,7 +67,7 @@ async function approveNameChange(payload: ApproveNameChangeService): Promise<Nam
     data: {
       status: NameChangeStatus.APPROVED,
       resolvedAt: new Date(),
-      reviewContext: undefined
+      reviewContext: PrismaTypes.DbNull
```

Comment:
Doesn't `null` work? Everything else looks great

Reply:
Sorry if it wasn't clear in my original phrasing, but when attempting to use `null` that was when I got the error I mentioned:
`Argument reviewContext for data.reviewContext must not be null. Please use undefined instead.`

I put the [documentation/reasoning](https://www.prisma.io/docs/orm/prisma-client/special-fields-and-types/working-with-json-fields#using-null-values) in my original post/description.

Reply:
Oh my bad, I misread the PR's description, sounds good to me